org.flightgear.FlightGear.yaml

```diff
@@ -17,7 +17,7 @@ rename-icon: flightgear
 finish-args:
 # shm + IPC are required for X11, otherwise the performances aren't great
 - --socket=x11
-- --device=shm
+- --device=all
```

Comment:
This is already included below.

Reply:
ah, using shm gave warning for me somehow

am using debian 10 flatpak-builder, perhaps is old version

Reply:
What was the warning message ? IIRC this shm thing is there for a while so I would be surprised if you didn't have it.

Reply:
unrecognized option for flag --device

the first succesful build runs well for me, so am leaving `--device=shm` as comment

Reply:
Ok, debian 10 has flatpak 1.2.5, which is two years old and doesn't include the shm device flag included in version 1.6.1, which is 11 months old. But it isn't a problem since the builders don't use debian so even if you can't build it locally with it, it should stay there as it will work anyway for most people. So please include it back.

Reply:
no worry, is done